jte-models/src/main/jte/dynamictemplates/kmethod.jte

```diff
@@ -4,7 +4,7 @@
 @param TemplateDescription template
 
     override fun ${Util.methodName(template)}(${Util.kotlinTypedParams(template)}): JteModel {
-        val paramMap = mapOf(
+        val paramMap = mapOf<String, Any?>(
```

Comment:
This is a minor fix after #282. When the map is empty, the type cannot be inferred. Having it explicitly declared here works for all the cases.